packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2585,6 +2657,7 @@ function flushSuspensePriorityWarningInDEV() {
           'feedback, and another update to perform the actual change.',
         // TODO: Add link to React docs with more information, once it exists
         componentNames.sort().join(', '),
+        componentsThatTriggeredSuspendNames.sort().join(', '),
```

Comment:
This array could be empty. In that case we should skip the "The components that triggered the update: " part of the message.